builder/builder-next/exporter/wrapper.go

```diff
@@ -52,6 +52,7 @@ func (e *imageExporterMobyWrapper) Resolve(ctx context.Context, id int, exporter
 	if _, has := exporterAttrs[string(exptypes.OptKeyDanglingPrefix)]; !has {
 		exporterAttrs[string(exptypes.OptKeyDanglingPrefix)] = "moby-dangling"
 	}
+	exporterAttrs[string(exptypes.OptKeyDanglingEmptyOnly)] = "true"
```

Comment:
Default behavior of the exporter was changed to create the dangling image always. This option was added to restore the previous behavior where it only tags it if there is no image name.